### PR TITLE
feat: Implement Simple Column container css style - EXO-61141 - Meeds-io/MIPs#37

### PR DESCRIPTION
**Pull request by sofyenne**
We will implement the css style for the newly added container `Simple Column Container`

### CODE REVIEW

File: platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiSimpleContainers/simpleColumnContainer.less [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.UIContainer.SimpleColumnContainer {
+  margin-right: 10%;
+}
+@media (max-width: 1319px) {
+  .UIContainer.SimpleColumnContainer {
```
Comment (azayati) [inline platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiSimpleContainers/simpleColumnContainer.less:6]: indentation

File: platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiSimpleContainers/simpleColumnContainer.less [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      margin-right: 20px;
+    }
+}
+@media (max-width: 979px) {
+.UIContainer.SimpleColumnContainer {
```
Comment (azayati) [inline platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiSimpleContainers/simpleColumnContainer.less]: indentation

File: platform-ui-skin/src/main/webapp/skin/less/component/Portal/UIEditInlineWS/Style.less [hunk 3]
```diff
@@ -153,6 +153,10 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 			background-position: 5px -401px;
                         height: 41px;
 		}
+		.SimpleColumnContainerLayout {
+			background-position: 5px -401px;
```
Comment (azayati) [inline platform-ui-skin/src/main/webapp/skin/less/component/Portal/UIEditInlineWS/Style.less:157]: to be verified with PO

**Review by azayati — CHANGES_REQUESTED** [review]

File: platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiSimpleContainers/responsiveColumnGroupContainer.less [hunk 4]
```diff
@@ -0,0 +1,10 @@
+.UIContainer.UIResponsiveColumnGroupContainer .UIRowContainer.flexColumnContainer {
+    display: flex;
+    flex-direction: row;
+}
```
Comment (azayati) [inline platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiSimpleContainers/responsiveColumnGroupContainer.less]:
```suggestion
.UIContainer.UIResponsiveColumnGroupContainer {
  .UIRowContainer.flexColumnContainer  {
    display: flex;
    flex-direction: row;
   } 
}
```

please format all less files

**Review by azayati — CHANGES_REQUESTED** [review]

File: platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiSimpleContainers/simpleColumnContainer.less (hunk 1 above)
Comment (sofyenne) [reply platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiSimpleContainers/simpleColumnContainer.less:6]: done !

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**